src/config.xml

```diff
@@ -275,11 +275,26 @@ Go to the <a href="commands.html">next</a> section or return to the
       <docs>
 <![CDATA[
  If the \c CREATE_SUBDIRS tag is set to \c YES then doxygen will create
- 4096 sub-directories (in 2 levels) under the output directory of each output
+ up to 4096 sub-directories (in 2 levels) under the output directory of each output
  format and will distribute the generated files over these directories.
  Enabling this option can be useful when feeding doxygen a huge amount of source
  files, where putting all generated files in the same directory would otherwise
  causes performance problems for the file system.
+ Adapt \c CREATE_SUBDIRS_LEVEL to control the number of sub-directories.
+]]>
+      </docs>
+    </option>
+    <option type='int' id='CREATE_SUBDIRS_LEVEL' minval='0' maxval='8' defval='8'>
+      <docs>
+<![CDATA[
+ Controls the number of sub-directories that will be create when \c CREATE_SUBDIRS tag is set to \c YES.
+ The number is computed by the formula 16 * (2 ^ CREATE_SUBDIRS_LEVEL).
```

Comment:
- see above more like a `\ref`
- `16 * (2 ^ CREATE_SUBDIRS_LEVEL)` should be more like a formula like `\f$16 * (2^{CREATE_SUBDIRS_LEVEL})\f$`